cypresse2e/cypress/integration/e2eTests/post/post.spec.js

```diff
@@ -0,0 +1,75 @@
+import HelpBoard from '../../../elements/pages/helpBoard'
+import Post from '../../../elements/pages/post'
+
+describe('FightPandemics Post for unauthorized user', () => {
+
+    const helpBoard = new HelpBoard();
+    const post = new Post();
+    const shareViaModalWindowTitle = "Share via..."
+ 
+
+    context('User opens Help Board', () => {
+        beforeEach(() => {
+            helpBoard.visit();
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking on Like button', () => {        
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                cy.get(post.getLikeButtonSelector()).click();
+                cy.validateCorrectScreenIsOpen("auth/login");
+            })                                                      
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking on Comment button', () => {     
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                cy.get(post.getCommentButtonSelector()).click();
+                cy.validateCorrectScreenIsOpen("auth/login");
+            })                                          
+        });
+
+        it('Unauthorized user can share a post. Share via... modal window is displayed', () => {  
```

Comment:
Here is just a simple note, no big deal. With this test we are checking if we are able to open the sharing window and we see everything relevant in it but don't actually test if we can share the post. (In the test description.)

Reply:
This is impossible:
https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs
Maybe we could include Puppeteer to do this?
I changed test description to "Unauthorized user can see a Share via... modal window"

Reply:
SSO is an issue with Cypress

Reply:
Here I had concerns only about the test description so it's ok.